gs-web-elasticsearch/doc/index.rst

```diff
@@ -162,6 +162,10 @@ The initial layer configuration panel for an Elasticsearch layer will include an
      - Name of the field
    * - ``Type``
      - Type of the field, as derived from the Elasticsearch schema. For geometry types, you have the option to provide a more specific data type.
+   * - ``order``
+     - Provides the option to return fields in a given order.

```

Comment:
Maybe indicate it's an integer so it's clearer how to use (e.g. "Integer order values are used to sort fields, where fields with smaller order are returned first").

Reply:
OK, thanks for the updated description, I'll include that.